corehq/apps/app_manager/tests/data/suite/sync_request.xml

```diff
@@ -0,0 +1,19 @@
+<partial>
+  <sync-request>
+    <post url="https://www.example.com/a/test_domain/phone/claim-case/">
+      <data key="case_id" ref="session/datum/case[@id]"/>
+      <data key="case_type" ref="session/datum/case/type"/>
+    </post>
+    <session>
+      <query url="https://www.example.com/a/test_domain/phone/search/">
+        <prompt key="name">
+          <display>Name</display>
```

Comment:
I'm sure this is supposed to look like ...

    <display>
        <text>
            <locale id="something.somethingelse"/>
        </text>
    </display>

... but I'm not sure what the somethings should be in this case. Where do I store what the app builder wants the label to be (and translations), and how do I refer to it here?